ADA/databricks_rstudio_personal_cluster.qmd

```diff
@@ -0,0 +1,110 @@
+---
+  title: "Setup Databricks personal compute cluster with RStudio"
+---
+
+<p class="text-muted">
+
+The following instructions set up an ODBC connection between your laptop and your DataBricks cluster, which can then be used in R/RStudio to query data using an ODBC based package.
```

Comment:
My only comment is that maybe we could add a bit more info about when/why you might want to do this, to differentiate it from the other method.